packages/database/drizzle/0000_military_bruce_banner.sql

```diff
@@ -11,6 +11,14 @@ CREATE TABLE IF NOT EXISTS "latitude"."commits" (
 	CONSTRAINT "commits_uuid_unique" UNIQUE("uuid")
 );
 --> statement-breakpoint
+CREATE TABLE IF NOT EXISTS "latitude"."convos" (
```

Comment:
can we name it conversations?

Reply:
don't you prefer convos? it's a common shorthand for conversations and we'll write less chars

Reply:
We don't write the name of the table ever again I think. This is only on the schema. I would prefer a more formal name

Reply:
I think it would be an unnecessary point of confusion so I'd prefer to keep consistent. conversations or convos everywhere, which one do u prefer? I don't have a strong opinion here.

Reply:
Whatever you prefer. I don't have any logical reason. Just feels more correct conversations